Stripe/PublicHeaders/STPPaymentCardTextField.h

```diff
@@ -144,6 +105,72 @@ IB_DESIGNABLE
 @property (nonatomic, readonly, nullable) UIImage *brandImage;
 
 /**
+ *  Whether or not the form currently contains a valid card number, 
+ *  expiration date, CVC, and postal code (if required).
+ *  @see STPCardValidator
+ */
+@property(nonatomic, readonly) BOOL isValid;
+
+/**
+ *  Enable/disable selecting or editing the field. Useful when submitting card details to Stripe.
+ */
+@property(nonatomic, getter=isEnabled) BOOL enabled;
+
+/**
+ *  The current card number displayed by the field. May or may not be valid, unless isValid is true, in which case it is guaranteed to be valid.
+ */
+@property(nonatomic, readonly, nullable) NSString *cardNumber;
+
+/**
+ *  The current expiration month displayed by the field (1 = January, etc). May or may not be valid, unless isValid is true, in which case it is guaranteed to be valid.
+ */
+@property(nonatomic, readonly) NSUInteger expirationMonth;
+
+/**
+ *  The current expiration month displayed by the field, as a string. This may or may not be a valid entry (i.e. "0", and may be 0-prefixed (i.e. "01" for January). You can use [STPCardValidator validationStateForExpirationMonth] to validate this value.
+ */
+@property(nonatomic, readonly, nullable) NSString *formattedExpirationMonth;
+
+/**
+ *  The current expiration year displayed by the field, modulo 100 (e.g. the year 2015 will be represented as 15). May or may not be valid, unless isValid is true, in which case it is guaranteed to be valid.
+ */
+@property(nonatomic, readonly) NSUInteger expirationYear;
+
+/**
+ *  The current expiration year displayed by the field, as a string. This is a 2-digit year (i.e. "15"), and may or may not be a valid entry. You can use [STPCardValidator validationStateForExpirationYear:inMonth] to validate this value.
+ */
+@property(nonatomic, readonly, nullable) NSString *formattedExpirationYear;
+
+/**
+ *  The current card CVC displayed by the field. May or may not be valid, unless isValid is true, in which case it is guaranteed to be valid.
+ */
+@property(nonatomic, readonly, nullable) NSString *cvc;
+
+/**
+ *  The current card ZIP or postal code displayed by the field.
+ */
+@property(nonatomic, readonly, nullable) NSString *postalCode;
+
+/**
+ *  Controls if a postal code entry field is displayed, and if so how its text
+ *  should be formatted and what input is valid.
+ *
+ *  Default is nil (no postal code entry is displayed).
+ *  Note that some non-nil values will still result in no postal code being
+ *  displayed (invalid country codes, or countries which do not commonly use
+ *  postal codes).
+ */
+@property(nonatomic, copy, nullable) NSString *countryCodeForPostalCodeFormattingAndValidation;
```

Comment:
(discussed in chat) can we update the API to be something like `postalCodeCollectionEnabled` + `countryCode` + use locale if `countryCode` is nil?